br/pkg/lightning/config/global.go

```diff
@@ -280,3 +312,41 @@ func LoadGlobalConfig(args []string, extraFlags func(*flag.FlagSet)) (*GlobalCon
 	cfg.App.Config.Adjust()
 	return cfg, nil
 }
+
+func extractCheckOnlyCfg(str string) (*CheckOnlyConfig, error) {
```

Comment:
How about implement `UnmarshalJSON` for `CheckOnlyConfig` like `PostOpLevel` and `CheckpointKeepStrategy`?

Reply:
not needed